scripts/qa_crowbarsetup.sh

```diff
@@ -3322,45 +3316,34 @@ function oncontroller_run_tempest
 {
     pushd /var/lib/openstack-tempest-test
     echo 1 > /proc/sys/kernel/sysrq
-    if iscloudver 5plus; then
-        if tempest help cleanup; then
-            tempest cleanup --init-saved-state
+    local tempestret
+
+    if iscloudver 6plus; then
+        tempest cleanup --init-saved-state
+        local opts=$tempestoptions
+        if iscloudver 7plus; then
+            ### backward compatibility, remove
+            opts=${opts/-s/--smoke}
```

Comment:
BTW the C7 gating runs didn't catch this because it was run with `tempestoptions="-t -s"`.